### DISCUSSION TITLE
Graduation Ceremony talent and mind wipe exploit

### DISCUSSION THREAD

**Discussion by Padex98** [FakeFishGames/Barotrauma#15140] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

WIth the assistant's perk "graduation ceremony", you can obtain all talents of all types of classes, by using the mind wipe syringe, since it doesn't remove the extra talents that aren't associated with the assistant role.

This gets exceptionally broken if the assistant obtains the "Family" perk from the "Captain's Skipper" talent tree, which essentially nullifies the point loss that Mind wipe should give you when you use it repeatedly.

So as long as you have enough mind wipe syringes, you can keep obtaining  the Graduation Ceremony perk until you've essentially unlocked every talent tree in the game.

In short, the mind wipe should remove the extra talents that the assistant would learn.

### Reproduction steps

1. obtain the Graduation ceremony perk
2. Use mind wipe
3. repeat until you unlock every extra talent possible. If you unlock the Family perk, you essentially get no penalty from using mind wipe
4. You become the ultimate barotrauma character

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/14858
